<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuiderWidget</class>
 <widget class="QWidget" name="GuiderWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>890</width>
    <height>580</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>890</width>
    <height>580</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>890</width>
    <height>580</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>866</width>
     <height>561</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="imageLayout">
      <item>
       <widget class="QScrollArea" name="imageScrollArea">
        <property name="minimumSize">
         <size>
          <width>650</width>
          <height>490</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>650</width>
          <height>490</height>
         </size>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>648</width>
           <height>488</height>
          </rect>
         </property>
         <widget class="QLabel" name="imageLabel">
          <property name="geometry">
           <rect>
            <x>3</x>
            <y>3</y>
            <width>640</width>
            <height>480</height>
           </rect>
          </property>
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="buttonFrame">
        <property name="maximumSize">
         <size>
          <width>650</width>
          <height>45</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QWidget" name="horizontalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>-1</x>
           <y>-1</y>
           <width>661</width>
           <height>51</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="buttonLayout">
          <item>
           <widget class="QPushButton" name="captureButton">
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Capture</string>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="calibrateButton">
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Calibrate</string>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="guideButton">
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Guide</string>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QGroupBox" name="exposureBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>90</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>75</height>
         </size>
        </property>
        <property name="title">
         <string>Exposure</string>
        </property>
        <widget class="QWidget" name="formLayoutWidget_6">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>30</y>
           <width>191</width>
           <height>69</height>
          </rect>
         </property>
         <layout class="QFormLayout" name="exposureLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::FieldsStayAtSizeHint</enum>
          </property>
          <property name="horizontalSpacing">
           <number>3</number>
          </property>
          <property name="verticalSpacing">
           <number>2</number>
          </property>
          <property name="topMargin">
           <number>2</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="timeLabel">
            <property name="text">
             <string>Time:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="binningLabel">
            <property name="text">
             <string>Binning:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="binningcomboBox"/>
          </item>
          <item row="0" column="1">
           <widget class="LogSpinBox" name="timeSpinbox"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="imageinfoBox">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>100</height>
         </size>
        </property>
        <property name="title">
         <string>Image Info</string>
        </property>
        <widget class="QWidget" name="formLayoutWidget_7">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>191</width>
           <height>80</height>
          </rect>
         </property>
         <layout class="QFormLayout" name="formLayout_3">
          <property name="topMargin">
           <number>4</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="maxLabel">
            <property name="text">
             <string>Maximum:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="minLabel">
            <property name="text">
             <string>Minimum:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="meanLabel">
            <property name="text">
             <string>Mean:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="maxField">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="meanField">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="minField">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="guidestarBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>100</height>
         </size>
        </property>
        <property name="title">
         <string>Guide star</string>
        </property>
        <widget class="QWidget" name="formLayoutWidget_8">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>182</width>
           <height>71</height>
          </rect>
         </property>
         <layout class="QFormLayout" name="guidestarLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::FieldsStayAtSizeHint</enum>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="starLabel">
            <property name="text">
             <string>Star:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QFrame" name="starFrame">
            <property name="minimumSize">
             <size>
              <width>110</width>
              <height>21</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QLineEdit" name="starxField">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>50</width>
               <height>21</height>
              </rect>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
            <widget class="QLabel" name="starcommaLabel">
             <property name="geometry">
              <rect>
               <x>53</x>
               <y>0</y>
               <width>5</width>
               <height>21</height>
              </rect>
             </property>
             <property name="text">
              <string>,</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="staryField">
             <property name="geometry">
              <rect>
               <x>60</x>
               <y>0</y>
               <width>50</width>
               <height>21</height>
              </rect>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="sizeLabel">
            <property name="text">
             <string>Size:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="sizeSpinbox">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="minimum">
             <number>16</number>
            </property>
            <property name="maximum">
             <number>64</number>
            </property>
            <property name="singleStep">
             <number>8</number>
            </property>
            <property name="value">
             <number>32</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="guideparametersBox">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>100</height>
         </size>
        </property>
        <property name="title">
         <string>Guide Parameters</string>
        </property>
        <widget class="QWidget" name="formLayoutWidget_9">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>181</width>
           <height>61</height>
          </rect>
         </property>
         <layout class="QFormLayout" name="guideparametersForm">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::FieldsStayAtSizeHint</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="focallengthLabel">
            <property name="text">
             <string>Focal Length:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="focallengthSpinbox">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="minimum">
             <number>100</number>
            </property>
            <property name="maximum">
             <number>3000</number>
            </property>
            <property name="singleStep">
             <number>100</number>
            </property>
            <property name="value">
             <number>600</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="guideintervalLabel">
            <property name="text">
             <string>Guide Interval:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="guideintervalSpinbox">
            <property name="minimumSize">
             <size>
              <width>65</width>
              <height>0</height>
             </size>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>30</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="monitoringBox">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>158</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>158</height>
         </size>
        </property>
        <property name="title">
         <string>Monitoring</string>
        </property>
        <widget class="QWidget" name="formLayoutWidget_10">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>181</width>
           <height>91</height>
          </rect>
         </property>
         <layout class="QFormLayout" name="formLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="guidingmonitorLabel">
            <property name="text">
             <string>Guiding Monitor:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="guidingmonitorButton">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Open</string>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="calibrationLabel">
            <property name="text">
             <string>Calibration:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="calibrationButton">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Open</string>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LogSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>logspinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>captureButton</sender>
   <signal>clicked()</signal>
   <receiver>GuiderWidget</receiver>
   <slot>capture()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>543</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>574</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>calibrateButton</sender>
   <signal>clicked()</signal>
   <receiver>GuiderWidget</receiver>
   <slot>calibrate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>410</x>
     <y>545</y>
    </hint>
    <hint type="destinationlabel">
     <x>410</x>
     <y>572</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>guideButton</sender>
   <signal>clicked()</signal>
   <receiver>GuiderWidget</receiver>
   <slot>guide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>589</x>
     <y>547</y>
    </hint>
    <hint type="destinationlabel">
     <x>589</x>
     <y>574</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>guidingmonitorButton</sender>
   <signal>clicked()</signal>
   <receiver>GuiderWidget</receiver>
   <slot>monitor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>850</x>
     <y>453</y>
    </hint>
    <hint type="destinationlabel">
     <x>882</x>
     <y>452</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>calibrationButton</sender>
   <signal>clicked()</signal>
   <receiver>GuiderWidget</receiver>
   <slot>calibration()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>843</x>
     <y>477</y>
    </hint>
    <hint type="destinationlabel">
     <x>880</x>
     <y>504</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>capture()</slot>
  <slot>calibrate()</slot>
  <slot>guide()</slot>
  <slot>monitor()</slot>
  <slot>calibration()</slot>
 </slots>
</ui>
